.gitignore

```diff
@@ -32,6 +32,7 @@ iterate.dat
 src/
 .settings
 .pods
+docker/mjkey.txt
```

Comment:
don't copy this into the repo. retrieve it from wherever it lives on the filesystem and then pass it as an environment variable. see Dockerfile.ci for an example.

Reply:
Done, now the mjkey is read from an environment variable instead.